docs/faq.md

```diff
@@ -0,0 +1,28 @@
+# Versions of the Readme Typing SVG
```

Comment:
If this is posed as a question, it could even be included below the FAQs header.
```suggestion
# How do I include Readme Typing SVG in my Readme?

Markdown files on GitHub support embedded images using Markdown or HTML.
```